python/meterpreter/ext_server_stdapi.py

```diff
@@ -2146,6 +2150,75 @@ def stdapi_net_config_get_routes(request, response):
         response += tlv_pack(TLV_TYPE_NETWORK_ROUTE, route_tlv)
     return ERROR_SUCCESS, response
 
+def windll_route_add_remove(is_add, request, response):
```

Comment:
These helper functions are typically `_win_route_add_remove` using the `_win` prefix for ones that are a bit more complicated than a thin wrapper around a Windll function like `windll_GetVersion`.

Reply:
thanks, will consider for next contributions.